data_model/clusters/ICDManagement.xml

```diff
@@ -55,18 +55,34 @@ Connectivity Standards Alliance
 508 Second Street, Suite 206
 Davis, CA 95616, USA
 -->
-<cluster xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd cluster cluster.xsd" id="0x0046" name="ICDManagement" revision="2">
+<cluster xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="types types.xsd cluster cluster.xsd" id="0x0046" name="ICDManagement" revision="1">
   <revisionHistory>
     <revision revision="1" summary="Initial Release"/>
-    <revision revision="2" summary="Addition of LIT ICD support and ActiveModeDuration"/>
```

Comment:
This PR is removing the revision update seems. Seem abnormal? @andy31415

Reply:
Maybe scraper bug? I did a spec "pull from master". Will check what I have.

Reply:
It is maybe because the adoc contains:

```adoc
[options="header",valign="middle"]
|===
| Revision | Description
| 1        | Initial Release
ifdef::in-progress,icd[]
| 2        | Addition of LIT ICD support, UserActiveModeTrigger, ActiveModeDuration, StayActiveResponse; removal of field Key in MonitoringRegistrationStruct
endif::[]
|===
```

I am not sure what to do with this ... this is a scraper & spec text issue.

Reply:
do we need the `in-progress` here? if SDK implements this and we never implemented version 1, why do we have an in-progress tag?

Reply:
Cluster was released in 1.2 with only a subset of attributes so from the spec point of view it makes sense for the in-progress...

For the scrapper, i guess we would need add in-progress fields for development?